<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="priceLineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>priceLineEdit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>nameLineEdit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="partComboBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addToCartButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>addToCartButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFromCartButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>290</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>removeFromCartButton</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="cartTextEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>104</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameToRemoveLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>nameToRemoveLineEdit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
